src/components/Table/Table.tsx

```diff
@@ -68,11 +68,17 @@ const TableRow = styled.tr`
   }
 
   @media (max-width: 768px) {
```

Comment:
should these breakpoints be hardcoded?

Reply:
Currently discussed with @gjones the max width is decided to be 768px for mobile
Should we create it as a token @gjones